src/RestEaseUnitTests/ImplementationBuilderTests/RawQueryParameterTests.cs

```diff
@@ -59,16 +65,43 @@ public void AddsRawQueryParam()
 
             implementation.FooAsync("&test=test2");
 
-            Assert.NotNull(requestInfo.RawQueryParameter);
-            Assert.Equal("&test=test2", requestInfo.RawQueryParameter.SerializeToString(null));
+            Assert.NotNull(requestInfo.RawQueryParameters);
+            Assert.Equal("&test=test2", requestInfo.RawQueryParameters.SerializeToString(null));
         }
 
         [Fact]
-        public void ThrowsIfTwoRawQueryParamsSpecified()
+        public void AddsTwoRawQueryParam()
         {
-            Assert.Throws<ImplementationCreationException>(() => this.builder.CreateImplementation<ITwoRawQueryStrings>(this.requester.Object));
+            var implementation = this.builder.CreateImplementation<ITwoRawQueryStrings>(this.requester.Object);
+            IRequestInfo requestInfo = null;
+
+            this.requester.Setup(x => x.RequestVoidAsync(It.IsAny<IRequestInfo>()))
+                .Callback((IRequestInfo r) => requestInfo = r)
+                .Returns(Task.FromResult(false));
+
+            implementation.FooAsync("&test=test2", "&test3=test4");
+
+            Assert.NotNull(requestInfo.RawQueryParameters);
+            Assert.Equal("&test=test2&test3=test4", requestInfo.RawQueryParameters.SerializeToString(null));
```

Comment:
I think we should be testing that `RawQueryParameters` contains 2 elements, then testing the values of those elements.